### PR TITLE
bump version to 2.5.0 and update release notes (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

File: RELEASE_NOTES [hunk 1]
```diff
@@ -3,6 +3,37 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
+v2.5.0 (December 17th 2015)
+---------------------------
+
+feature + bugfix release
+- add support for IBM XL compilers on Power7 and PowerPC (BlueGene) (#1470)
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: would be nice to document which easyconfig names correspond to that; other than that, it all LGTM!

Comment (boegel) [inline RELEASE_NOTES]: None yet, only the support to use the IBM XL compilers is there for now in the framework, the rest is WIP (cfr. https://github.com/hpcugent/easybuild-easyblocks/pull/750)

**Comment by wpoely86** [comment]
:+1:

**Comment by boegel** [comment]
Going in, thanks for the review @fgeorgatos and @wpoely86!

**Pull request merged**